packages/react-menu/src/components/MenuGroup/useMenuGroupStyles.ts

```diff
@@ -0,0 +1,10 @@
+import { mergeClasses } from '@fluentui/react-make-styles';
+import type { MenuGroupState } from './MenuGroup.types';
+
+export const menuGroupClassName = 'fui-MenuGroup';
+
+export const useMenuGroupStyles = (state: MenuGroupState): MenuGroupState => {
+  state.root.className = mergeClasses(menuGroupClassName, state.root.className);
```

Comment:
Some components does not have `use*Styles` hook, but I decided to add it to follow the same pattern rather than merging classes in the state.